libbeat/processors/processor.go

```diff
@@ -71,7 +71,12 @@ func New(config PluginConfig) (*Processors, error) {
 
 		gen, exists := registry.reg[actionName]
 		if !exists {
-			return nil, errors.Errorf("the processor action %s does not exist", actionName)
+			var validActions []string
```

Comment:
How is this change related?

Reply:
I made a mistake in earlier versions of this and found this debugging info useful. Glad to move it to a new PR if you feel it's worthwhile.

Reply:
LGTM.